libs/coin-bitcoin/src/js-getTransactionStatus.ts

```diff
@@ -72,12 +72,10 @@ const getTransactionStatus = async (a: Account, t: Transaction): Promise<Transac
     );
   }
 
-  const sumOfInputs = (txInputs || []).reduce(
-    (sum, input) => sum.plus(input.value),
-    new BigNumber(0),
-  );
+  //TODO: check if it is ok in case of `undefined` value, a zero is ok
```

Comment:
I think it is OK. 
"undefined" is not supposed to occur in input.value unless there's a bug. We can consider “undefined” as 0